build.gradle.kts

```diff
@@ -0,0 +1,80 @@
+import net.ltgt.gradle.errorprone.errorprone
+
+plugins {
+    java
+    alias(libs.plugins.spring.boot)
+    alias(libs.plugins.spring.dependency.management)
+    jacoco
+    alias(libs.plugins.errorprone)
```

Comment:
is there an opportunity to align the coding standards here with what we use in the main Solr project?   I'm thinking errorprone and spotless etc?